### PR TITLE
Resolve final review comments

### PR DESCRIPTION
**Pull request by calmbeing**
### Description

add a description of your changes here...

### Rationale

tell us why we need these changes...

### Example

add an example CLI or API response...

### Changes

Notable changes: 
* add each change in a bullet point here
* ...

### CODE REVIEW

File: consensus/parlia/parlia.go [hunk 1]
```diff
@@ -1159,46 +1159,46 @@ func (p *Parlia) WithinValidatorSet(chain consensus.ChainHeaderReader, header *t
 	}
 	validators := snap.Validators
 	if _, ok := validators[p.val]; ok {
-		return true
+		return ok
```
Comment (NashBC) [inline consensus/parlia/parlia.go:1162]: I think you misunderstood the comment.

```
_, ok := validators[p.val]
return ok
```

File: core/vote/vote_manager.go [hunk 2]
```diff
@@ -136,11 +136,11 @@ func (voteManager *VoteManager) loop() {
 				}
 				if err := voteManager.journal.WriteVote(voteMessage); err != nil {
 					log.Warn("Failed to write vote into journal", "err", err)
-					votesJournalErrorMetric(vote.TargetNumber, vote.TargetHash).Inc(1)
+					votesJournalErrorMetric().Inc(1)
 					continue
 				}
 
-				log.Info("vote manager produced vote", "voteHash", voteMessage.Hash())
+				log.Info("vote manager produced vote", "votedBlockNumber", voteMessage.Data.TargetNumber, "votedBlockHash", voteMessage.Data.TargetHash, "voteMessageHash", voteMessage.Hash())
```
Comment (NashBC) [inline core/vote/vote_manager.go:143]: Can we change this log level to debug?

Comment (calmbeing) [reply core/vote/vote_manager.go]: Done

File: consensus/parlia/parlia.go (hunk 1 above)
Comment (calmbeing) [reply consensus/parlia/parlia.go]: Done

**Pull request merged**